### PR TITLE
Destructure params

### PR DESCRIPTION
**Pull request by alibombaye**

### CODE REVIEW

File: app/manifestProvider.js [hunk 1]
```diff
@@ -7,7 +7,9 @@ export class ManifestProvider {
     return this.dashboardManifest;
   }
 
-  getSubDashboardManifest(dashboardId) {
+  getSubDashboardManifest({
+    dashboardId,
```
Comment (BenBJSS) [inline app/manifestProvider.js]: Move to same line

File: app/routes.js [hunk 2]
```diff
@@ -112,11 +112,13 @@ router.get('/solution/:solutionId/preview', async (req, res, next) => {
...
   const { solutionId } = req.params;
   try {
-    const response = await postSubmitForModeration(solutionId);
+    const response = await postSubmitForModeration({ solutionId });
     if (response.success) {
       res.redirect(`/solution/${solutionId}`);
     } else {
-      const context = await getMarketingPageDashboardContext(solutionId, response);
+      const context = await getMarketingPageDashboardContext(
+        { solutionId, validationErrors: response },
```
Comment (BenBJSS) [inline app/routes.js]: Match style

File: app/manifestProvider.js (hunk 1 above)
Comment (alibombaye) [reply app/manifestProvider.js]: done

File: app/routes.js (hunk 2 above)
Comment (alibombaye) [reply app/routes.js]: done

**Pull request merged**